ui/app/src/model/project-client.ts

```diff
@@ -31,8 +31,57 @@ const dependingResources = [dashboardResource, variableResource, datasourceResou
 
 type ProjectListOptions = Omit<UseQueryOptions<ProjectResource[], Error>, 'queryKey' | 'queryFn'>;
 
+export interface ProjectDashboardsGroup {
```

Comment:
You put in the client something that is purely for the view. I would say that this model and the associated use should actually be moved the unique place they are used.

Reply:
Forget about that let's keep it there